tests/unit_tests/test_FaServerThread.py

```diff
@@ -75,3 +75,72 @@ def test_command_game_host_calls_host_game_invalid_title(connected_socket,
     server_thread.command_game_host(test_game_info_invalid)
     assert mock_lobby_server.games.create_game.mock_calls == []
     server_thread.sendJSON.assert_called_with(dict(command="notice", style="error", text="Non-ascii characters in game name detected."))
+
+# ModVault
+
+# TODO: find a better way
+OneTimeTrue = False
+def one_time_true():
+    global OneTimeTrue
+    if OneTimeTrue:
+        return False
+    OneTimeTrue = True
+    return True
+
+@mock.patch('src.lobbyconnection.QSqlQuery')
+@mock.patch('src.lobbyconnection.Config')
+def test_mod_vault_start(mock_config, mock_query, connected_socket, mock_lobby_server):
+    server_thread = LobbyConnection(connected_socket, mock_lobby_server)
+    server_thread.sendJSON = mock.Mock()
```

Comment:
I am not able to use a fixture: 

```@pytest.fixture
def my_server_thread(connected_socket, mock_lobby_server):
    return LobbyConnection(connected_socket, mock_lobby_server)```
I get:

![image](https://cloud.githubusercontent.com/assets/1217999/6891322/df55b8c8-d6b9-11e4-8dac-b664a2dc2152.png)